index.js

```diff
@@ -43,15 +29,15 @@ module.exports = () => {
       }
 
       if (type === 'IPFS_INIT_FINISHED') {
-        return Object.assign({}, state, { ready: true, identity: payload })
+        return Object.assign({}, state, { ready: true, ...payload })
```

Comment:
I don't think we can use spread without adding a transpilation step to this repo